openapi/openapi.yaml

```
@@ -148,3 +148,54 @@ components:
         x-facker: internet.url
         example:
           https://example.com/api/v1/recambios/
+  
+  parameters:
+    Nombre_Recambio:
+      name: Nombre_Recambio
+      in: path
+      description: Nombre del recambio correspondiente a un articulo del taller
+      required: true
+      schema:
+        $ref: '#/components/schemas/Nombre_Recambio'
+
+    Proveedor_Recambio:
+      name: Proveedor_Recambio
+      in: path
+      description: El proveedor de un recambio para el taller de vehículos
+      required: false
```

```suggestion
      required: true
```
Todos los parametros que tengan un "in: path" segun la documentación de swagger se requieren.
https://swagger.io/docs/specification/describing-parameters/